### PR TITLE
[micro][bot] Telegram Messenger support

### PR DESCRIPTION
**Pull request by Scukerman**
```
$ micro bot --inputs=telegram --telegram_token=BOT_TOKEN --telegram_whitelist=COMMA_SEPARATED_NICKNAMES_WITHOUT_AT_SYMBOL
```

### CODE REVIEW

File: micro/bot/input/telegram/conn.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package telegram
+
+import (
+	"gopkg.in/telegram-bot-api.v4"
```
Comment (asim) [inline micro/bot/input/telegram/conn.go]: please separate external imports from internal by placing them below with a new line

example
```
import (
    "errors"

    "github.com/micro/micro/bot/input"
)
```

**Comment by asim** [comment]
Has this bot input been tested?

File: micro/bot/input/telegram/conn.go (hunk 1 above)
Comment (Scukerman) [inline micro/bot/input/telegram/conn.go]: Done. Made it in single one commit.

**Comment by Scukerman** [comment]
Sure I did. If it hasn't, then why in the world I would made a PR? :)

**Comment by asim** [comment]
Ah you never know :)

**Comment by asim** [comment]
Mind just adding a little readme as well to make it easy for people. Like the discord one https://github.com/micro/go-plugins/tree/master/micro/bot/input/discord

**Comment by asim** [comment]
Thanks

**Pull request merged**